aksel.nav.no/website/pages/eksempler/table/sortable.tsx

```diff
@@ -19,15 +23,19 @@ const Example = () => {
     );
   };
 
-  const comparator = (a, b, orderBy) => {
-    if (b[orderBy] < a[orderBy] || b[orderBy] === undefined) {
+  function comparator<T>(a: T, b: T, orderBy: keyof T): number {
+    if (
+      b[orderBy] === null ||
+      b[orderBy] === undefined ||
```

Comment:
We _could_ just do a nullish check by doing `b[orderBy] == null` (checks for both null and undefined), but it might be unclear.

Reply:
That works 👍